src/main/java/org/jenkinsci/plugins/ghprb/GhprbRepository.java

```diff
@@ -52,6 +52,9 @@
 
     private static final transient EnumSet<GHEvent> HOOK_EVENTS = EnumSet.of(GHEvent.ISSUE_COMMENT, GHEvent.PULL_REQUEST);
 
+    private static final transient boolean INSECURE_WEBHOOKS = Boolean.parseBoolean(
+            System.getProperty("jenkins.ghprb.webhook.insecure", System.getenv("jenkins.ghprb.webhook.insecure")));
```

Comment:
There is a Jenkins helper class for this kind of stuff which has a bunch of error handling.  I’ll try to find it and comment again.

Reply:
Sure! I'll be happy to update the PR.

Reply:
Rely on `jenkins.util.SystemProperties`.  See an example in #543.  I would recommend naming the system property after the class which is the same as in #543.  Other Jenkins properties do that.  For a list of available methods [on `jenkins.util.SystemProperties` refer to the javadoc](
http://javadoc.jenkins.io/jenkins/util/SystemProperties.html).  I imagine you’ll want the `getBoolean()` method.

Reply:
Example,

```java
SystemProperties.getBoolean(GhprbRepository.class.getName() + ".insecure", false)
```

Note: I set the default to false if an admin does not define the property because that is the current behavior.

Reply:
Defining on the CLI property would be:

```java
java ... -Dorg.jenkinsci.plugins.ghprb.GhprbRepository.insecure=true ...
```

@Ladicek thanks for taking the time to contribute this.  I really appreciate your effort.

Reply:
I'll take a look and update the PR, thanks!

Reply:
I updated the PR with this suggestion. Sorry for the delay, I was on vacation :-)